main/res/values/strings.xml

```diff
@@ -840,6 +843,44 @@
+    <string name="contentstorage_selectfolder_dialog_choice_nocopy">Do not copy or move item(s)</string>
+    <string name="contentstorage_selectfolder_dialog_choice_copy">Copy items(s)</string>
+    <string name="contentstorage_selectfolder_dialog_choice_move">Move item(s)</string>
+    <string name="contentstorage_selectfolder_dialog_copy_move_finished_title">Folder %s: Copy/Move done</string>
+    <string name="contentstorage_selectfolder_dialog_copy_move_finished_msg_html"><![CDATA[<p>Copy/Move had result %s. %d file(s) and %d subdirectories were copied/moved.</p><p>Click \'ok\' finalize the directory change.</p>]]></string>
+    <string name="contentstorage_folder_selection_aborted">Aborting folder selection, folder is \'%s\'</string>
+    <string name="contentstorage_folder_selection_failed">Selected folder not accessible, aborting selection. Folder is \'%s\'</string>
+    <string name="contentstorage_folder_selection_success">Folder set to \'%s\'</string>
+    <string name="contentstorage_file_selection_aborted">Aborting file selection</string>
+    <string name="contentstorage_file_selection_success">File selected: \'%s\'</string>
+    <string name="contentstorage_err_publicfolder_inaccessible_falling_back">Folder %s: \'%s\' is not accessible, falling back to \'%s\'</string>
+    <string name="contentstorage_err_read_failed">File \'%s\' could not be read, see logfile for details</string>
+    <string name="contentstorage_err_write_failed">File \'%s\' could not be written, see logfile for details</string>
+    <string name="contentstorage_err_create_failed">File \'%s\' in \'%s\' could not be created, see logfile for details</string>
+    <string name="contentstorage_err_delete_failed">File \'%s\' could not be deleted, see logfile for details</string>
+    <string name="contentstorage_err_list_failed">Files in Folder \'%s\' could not be listed, see logfile for details</string>
+    <string name="contentstorage_err_folder_access_failed">Folder \'%s\' could not be accessed, see logfile for details</string>
+    <string name="contentstorage_err_copy_failed">File \'%s\' could not be copied/moved to \'%s\' (move=%s), see logfile for details</string>
+
+    <!-- Persistable Folders -->
+    <string name="persistablefolder_usertype_userdefined">User-defined</string>
+    <string name="persistablefolder_usertype_default">Default</string>
+
+
+    <string name="persistablefolder_base">Base</string>
+    <string name="persistablefolder_offline_maps">Offline Maps</string>
+    <string name="persistablefolder_offline_maps_themes">Map Themes</string>
+    <string name="persistablefolder_gpx">GPX</string>
+    <string name="persistablefolder_backup">Backup</string>
+    <string name="persistablefolder_logfiles">Logfiles</string>
+    <string name="persistablefolder_fieldnotes">Field Notes</string>
+    <string name="persistablefolder_images">Log Images</string>
+
```

Comment:
Are these `persistablefolder_*` only used for the system information? In that case we don't need localization.

Reply:
Iirc also in the folder selection dialog title. But I have to check

Reply:
Confirmed. It is used in the "select folder" dialogs like this:
![image](https://user-images.githubusercontent.com/6909759/104299083-0d198980-54c5-11eb-95e3-1f64bb392740.png)

Found no other places though.

Do you think that a non-translatable value is enough for this case?

Reply:
No, in that case I think it should be translated. So all fine here.